scss/mixins/_decoration.scss

```diff
@@ -2,12 +2,12 @@
 @mixin fill( $color: null, $bg: null, $border: null, $gradient: null ) {
     border-color: $border;
     color: $color;
-    background-color: $bg;
+    background: $bg;
```

Comment:
@gyoshev This wasn't in the original pull request, but I just realized that it would have introduced a bug if this didn't get changed as well. I'm not sure if this is something you would want to change / include in the theme - adds a lot of flexibility to the `@fill` mixin but this may introduce some issues where it is used elsewhere.